tests/testthat/test-read_resource.R

```diff
@@ -662,3 +662,62 @@ test_that("read_resource() handles other types", {
   # Guess undefined types, unknown types are blocked by check_schema()
   expect_type(resource$no_type, "logical")
 })
+
+test_that("read_resource() allows selecting of resource columns", {
+  # single column
+  expect_named(
+    read_resource(example_package,
+                  "observations",
+                  col_select = "observation_id"),
+    "observation_id")
+  # multiple columns
+  expect_named(
+    read_resource(
+      example_package,
+      "observations",
+      col_select = c("observation_id", "deployment_id")
+    ),
+    c(
+      "observation_id",
+      "deployment_id"
+    )
+  )
+  # different order
+  expect_named(
+    read_resource(example_package,
+                  "observations",
+                  col_select = c("observation_id","scientific_name","deployment_id")),
```

Comment:
Use spaces after comma

Reply:
Forgot to lint, fixed: e4b7702e1c06e779c85dcbbda9dd3319bcfa43ac